src/validations.ts

```diff
@@ -12,19 +12,27 @@ import {
   SafeSellNowRequest,
   SellNowRequest
 } from './types/requests'
-import { Order, OrderType } from './types/subgraph'
-import { Address, isAddress } from 'viem'
+import { Address, isAddress, isHex } from 'viem'
 
-const addressSchema = Joi.string().custom((value, helpers) => {
+export const hashSchema = Joi.string().custom((value, helpers) => {
+  if (!isHex(value)) {
```

Comment:
I prefer using viem functions. If anything changes this will be updated when we update viem. Under the hood is doing the same thing (checking if it starts with 0x). It also has the option to se {string: true} and it checks for hex only chars